py-polars/tests/unit/datatypes/test_enum.py

```diff
@@ -195,14 +195,6 @@ def test_extend_to_an_enum() -> None:
     assert s.null_count() == 1
 
 
-def test_series_init_uninstantiated_enum() -> None:
```

Comment:
This test should still be valid right? We do not allow creating a series with an empty `Enum` type, the None is just a placeholder for all Enums